### ISSUE TITLE
calx doesn't work with "number" fields

### ISSUE THREAD

**Issue by cyphix333** [xsanisty/jquery-calx#31]
Not sure if it is a bug or it just hasn't been implemented yet but I was trying to use calx with _number_ fields where you can use the up/down arrows but it doesn't seem to support them as whenever the number field lost focus it would reset the number field back to "0". I changed to standard "text" field and it worked fine.

**Comment by UsmanShahzad** [comment]
Yes this bug still seems to be there, if you enter the value manually it works but not with spinner.
